quality_of_life_changer.client/src/components/profile/calendars-card/calendars-card.js

```diff
@@ -0,0 +1,24 @@
+import { Component } from "react";
+import { Button, Card } from "react-bootstrap";
+
+class CalendarCard extends Component {
+  render() {
+    const copy = async () => {
```

Comment:
don't create any functions inside `render()`, you can create a handler function in component itself